bundles/action/org.openhab.action.telegram/src/main/java/org/openhab/action/telegram/internal/TelegramActionService.java

```diff
@@ -72,4 +83,8 @@ public void updated(Dictionary<String, ?> config) throws ConfigurationException
             isProperlyConfigured = true;
         }
     }
+
+    private String getConfigValue(Dictionary<String, ?> config, String key) {
```

Comment:
This method provides no value and should be removed.

You're also using the Objects.toString method incorrectly, as you've omitted the second `null` parameter.

Reply:
This one is fixed. I thought that the default value (if none is given) is `null` but it is `"null"` as String.